lib/gui/components/drive-selector/controllers/drive-selector.js

```diff
@@ -170,4 +170,26 @@ module.exports = function(
     });
   };
 
+  /**
+   * @summary Get drive and image compatibility status object
+   * @function
+   * @public
+   *
+   * @description
+   * Given an image and a drive, return their compatibility status object
+   * containing the status type (ERROR, WARNING, and OK), and accompanying
+   * status message.
+   * Almost just a synonym of its namesake, the difference is that the image
+   * argument is implicit here for the sake of shortening.
+   *
+   * @param {Object} drive - drive
+   * @returns {Object} compatibility status object
+   *
+   * @example
+   * let { type, message } = DriveSelectorController.getDriveImageCompatibilityStatus(drive);
```

Comment:
Should we encourage the use of `const` here, for pure functional programming's sake?